### DISCUSSION TITLE
Calling Pluto on an existing notebook

### DISCUSSION THREAD

**Discussion by NiallPalfreyman** [fonsp/Pluto.jl#1741] [Questions about using Pluto] [answered]
Hi,

I really like Pluto, and am looking for ways to integrate it into my other courseware. Hence my question: Is there any way of calling Pluto directly on an existing notebook? So, for example, in the middle of one of my own courses, would it be possible to make a call something like: ```Pluto.run("CuteScience.jl")``` ?

Thanks and best wishes,
Niall.

**Comment by fonsp** [comment, marked as answer]
Try `Pluto.run(notebook="CuteScience.jl")`

**Reply by NiallPalfreyman** [reply]
That's it - perfect! Thanks!
